pkg/operator/workload/manage.go

```diff
@@ -0,0 +1,288 @@
+package workload
+
+import (
```

Comment:
Having code moves and real changes in the same commit is bad. Nobody can review this in a sensible way.

Reply:
Indeed, I need to redo the commits on this branch and make sure the code move is addressed.